components/resourcesAdmin/ProjectComponentStyles.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+
+const HEADER_COLOUR = "#1e2530";
+const HEADER_HEIGHT = "6vh";
```

Comment:
I think it might be better to abstract this into a file in the `util` section instead of creating a new file for `resourcesAdmin` - maybe `utils/admin`?